app/assets/stylesheets/components/_slot-picker.scss

```diff
@@ -0,0 +1,351 @@
+$calendar-border: $grey-3;
+$calendar-selection: $light-blue-50;
+$calendar-today: $light-blue;
+$calendar-hover: $light-blue-25;
+$calendar-busy-background-thin-stripes: $color-alto;
+$calendar-busy-background-wide-stripes: $color-grey-alabaster;
+$calendar-day-row-background: $color-grey-alabaster;
+$calendar-confirmed-slot-border: $grey-2;
```

Comment:
Although these are handy here, convention for this project has been to define these in https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/assets/stylesheets/lib/_variables.scss